From 99b8373dbe8fb3bac835ebad90f91cfd213e4acb Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Wed, 15 Mar 2017 10:02:54 -0300
Subject: [PATCH] Forward EGL cflags into epoxy.pc

When building mesa egl without x11 and gles2 the headers need a
MESA_EGL_NO_X11_HEADERS define to avoid including X11 headers.
Forward EGL CFLAGS into epoxy.pc to avoid build failures when linking
against libepoxy on this scenario.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
Status: pull request https://github.com/anholt/libepoxy/pull/113

 epoxy.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/epoxy.pc.in b/epoxy.pc.in
index 7828a77..8278b7d 100644
--- a/epoxy.pc.in
+++ b/epoxy.pc.in
@@ -10,6 +10,6 @@ epoxy_has_wgl=@epoxy_has_wgl@
 Name: epoxy
 Description: epoxy GL dispatch Library
 Version: @PACKAGE_VERSION@
-Cflags: -I${includedir}
+Cflags: -I${includedir} @EGL_CFLAGS@
 Libs: -L${libdir} -lepoxy
 Libs.private: @DLOPEN_LIBS@
-- 
2.10.2

